support-workers/src/test/scala/com/gu/support/workers/lambdas/CreatePaymentMethodSpec.scala

```diff
@@ -70,6 +75,53 @@ class CreatePaymentMethodSpec extends AsyncLambdaSpec with MockContext {
       }
   }
 
+  it should "retrieve a valid DirectDebitPaymentMethod when given valid DD fields" in {
+
+    val createPaymentMethod = new CreatePaymentMethod(mockServices)
+
+    val outStream = new ByteArrayOutputStream()
+
+    createPaymentMethod
+      .handleRequestFuture(wrapFixture(createDirectDebitDigitalPackJson), outStream, context)
+      .map { _ =>
+        // Check the output
+        val createSalesforceContactState = Encoding.in[CreateSalesforceContactState](outStream.toInputStream)
+
+        createSalesforceContactState.isSuccess should be(true)
+        createSalesforceContactState.get._1.paymentMethod match {
+          case dd: DirectDebitPaymentMethod =>
+            withClue("direct debit: " + dd) {
+              dd.PaymentGateway should be(DirectDebitGateway)
+            }
+            dd.Email should be(None)
```

Comment:
minor but if the clue is useful we can wrap the email check in it too.  same in the below test
```suggestion
            withClue("direct debit: " + dd) {
              dd.PaymentGateway should be(DirectDebitGateway)
              dd.Email should be(None)
            }
```